herddb-core/src/main/java/herddb/server/ServerConfiguration.java

```diff
@@ -151,6 +151,9 @@
     public static final String PROPERTY_CHECKPOINT_PERIOD = "server.checkpoint.period";
     public static final long PROPERTY_CHECKPOINT_PERIOD_DEFAULT = 1000L * 60 * 15;
 
+    public static final String PROPERTY_ENABLE_CONSISTENCY_CHECK = "server.localconsistencycheck.enabled";
```

Comment:
This should be added on the example configuration file too

Reply:
We will drop this feature as soon as we have the distributed consistency checker.
I think we should not advertise it too much

Reply:
Good point